src/read_utils.rs

```diff
@@ -17,15 +17,11 @@ use crate::{
     objects::{self, Contributor},
```

Comment:
Since this file now only contains things about the configuration, how about renaming it `config.rs` or `configuration.rs`?